src/resources/ledstrip.rs

```diff
@@ -0,0 +1,49 @@
+use std::collections::HashSet;
+
+use super::color::Color;
+pub type EffectInterval = (usize, usize);
+#[derive(Debug)]
+pub struct LedStrip {
+    pub size: usize,
+    pub colors: Vec<Color>,
+    pub effects: Vec<(i32, EffectInterval)>,
+    used_led_count: usize,
+}
+
+impl LedStrip {
+    pub fn new() -> LedStrip {
```

Comment:
ecq on peut juste impl `Default`

Reply:
Ouais, fixed. Online ca dit quon devrait mm avoir les 2. Pour le moment jai juste remove new() et garder Default pcq ca donne un lint error de dead code. (https://stackoverflow.com/questions/41510424/most-idiomatic-way-to-create-a-default-struct)